aspnetcore-webapi/UsingResultPatternInNETWebAPI/UsingResultPatternInNETWebAPI/v5/Controllers/FluentResultsContactsController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Net;
+
+namespace UsingResultPatternInNETWebAPI.v5.Controllers;
+
+[ApiController]
+[Route("api/v5/contacts")]
+public class FluentResultsContactsController : ControllerBase
+{
+    private readonly FluentResultsContactsService _contactsService;
+
+    public FluentResultsContactsController(FluentResultsContactsService contactsService)
+    {
+        _contactsService = contactsService;
+    }
+
+    [HttpGet]
+    [ProducesResponseType(200)]
+    public ActionResult<IEnumerable<ContactDto>> GetAll()
+    {
+        var result = _contactsService.GetAll();
+
+        if (result.IsFailed)
```

Comment:
For GetAll() method I don't see any case when this block of code will be executed.

Reply:
It can be used for testing (there is a test case in Api.http for that), given a user clones this repo and wants to verify that the API is working as expected. I didn't use it in the article as it is pretty long already. Same question - should we remove it?

Reply:
We are returning Result.Ok() from GetAll() method? In which scenario, result.IsFailed will be true?

Reply:
You are right. It was added artificially to keep it consistent.